app/api.py

```diff
@@ -65,30 +74,15 @@ def create_warrior():
     # Check the name is more than 100 characters
     if len(name) > 100:
         return jsonify({"message": "Bad Request - name is too long"}), 400
+
     # Check for empty skills
-    if fight_skills == None:
+    if fight_skills is None:
         return jsonify({"message": "Bad Request - fight_skills cannot be empty"}), 400
+    
     # Check for more than 20 fight skills
-    try:
-        if len(fight_skills) > 20:
-            return (
-                jsonify(
-                    {
-                        "message": "Bad Request - fight_skills cannot cannot have more than 20 skills"
-                    }
-                ),
-                400,
-            )
-    except Exception:
-        return jsonify({"message": "fight skill is not a string"}), 400
-    # Check for skills that are more than 250 characters
-    if any(len(fight_skill) > 250 for fight_skill in fight_skills):
-        return jsonify({"message": "Bad Request - a fight skill name is too long"}), 400
-    # Check for total sum of the fight skill character lengths doesn't exceed max length
-    max_skills_length = 5019
-    if sum([len(fight_skill) for fight_skill in fight_skills]) > max_skills_length:
-        return jsonify({"message": "Bad Request - fight skills length exceeded"}), 400
-
+    if len(fight_skills) > 20:
+        return jsonify({"message": "Bad Request - fight_skills cannot have more than 20 skills"}), 400
+        
```

Comment:
After the latest `main` commit merge into this branch, there are still duplication fight_skill validations.

Reply:
I have already removed this validations from this part but it see them still in the main. Can I remove them from the main too and push the changes? I am not sure how it works